resources/views/entities/components/profile/races.blade.php

```diff
@@ -10,6 +10,19 @@
         <i class="fa-solid fa-chevron-down"></i>
         {{ __('crud.tabs.profile') }}
     </div>
+    @if (!$model->locations->isEmpty())
+            <div class="element profile-location">
+                <div class="title">{{ __('races.fields.locations') }}</div>
```

Comment:
Variable name needs fixing

Reply:
What variable?

Reply:
Woops, this was for another area in the code, but looks fixed 🙈